src/types/router.ts

```diff
@@ -19,6 +20,7 @@ export type Router<
   resolve: RouterResolve<TRoutes>,
   push: RouterPush<TRoutes>,
   replace: RouterReplace<TRoutes>,
+  find: RouterFind<TRoutes>,
```

Comment:
Since this utility returns ResolvedRoute types, do we even want to add this to our public api? That would mean ResolvedRoute is public and we'd have to make any changes to it backwards compatible. I'm hoping ResolvedRoute can be completely internal

Reply:
Well I guess this would just return RouterRoute actually 🤔 ignore me